testkit/scaladsl/src/main/scala/com/lightbend/lagom/scaladsl/testkit/ServiceTest.scala

```diff
@@ -114,6 +123,15 @@ object ServiceTest {
     @ApiMayChange
     def withSsl(enabled: Boolean): Setup
 
+    /**
+     * Enable or disable SSL on the specified port.
+     * @param enabled True if the server should bind an HTTP+TLS port, or false if only HTTP should be bound.
+     * @param port The port to bind to.
+     * @return A copy of this setup.
+     */
+    @ApiMayChange
+    def withSsl(enabled: Boolean, port: Int): Setup
```

Comment:
these were duplicated for backwards compatibility. but we could also add the parameter to the existing method with a default value.

Reply:
It's better to overload the method like you did. Let's keep it so.